### ISSUE TITLE
refresh-token not supported on Django's end

### ISSUE THREAD

**Issue by eranzim** [akveo/ngx-admin-bundle-support#54]
Hi!
After a few minutes in the frontend, the token should (and does) time-out, but then the refresh token should kick in and allow more time on the same token, so that the user doesn't need to log back in every 5 minutes. The frontend code indeed sends these refresh-token requests, but there is no code in the Django backend to handle these HTTP requests... This makes it impossible to use this bundle in production.

**Comment by developer-cormatix** [comment]
yes, Refresh-token code is missing. and basic api should provide refresh token feature.
